ceph_installer/controllers/mon.py

```diff
@@ -54,14 +54,17 @@ def configure(self):
     @configure.when(method='POST', template='json')
     @validate(schemas.mon_configure_schema, handler="/errors/schema")
     def configure_post(self):
-        hosts = [request.json['host']]
+        hosts = ["{} monitor_interface={}".format(request.json['host'], request.json["monitor_interface"])]
```

Comment:
I just thought it'd be clunky to construct a dict in the format ``parse_monitors`` needed. But, I suppose I could reuse ``parse_monitors`` here.